crest/Assets/Crest/Crest/Scripts/LodData/RegisterLodDataInput.cs

```diff
@@ -52,6 +52,10 @@ public abstract partial class RegisterLodDataInputBase : MonoBehaviour, ILodData
         public abstract bool Enabled { get; }
 
         public static int sp_Weight = Shader.PropertyToID("_Weight");
+        public static int sp_DisplacementAtInputPosition = Shader.PropertyToID("_DisplacementAtInputPosition");
+
+        [SerializeField, Tooltip("Correct for horizontal displacement so that this input does not move from side to side with the waves.")]
+        bool _applyDisplacementCorrection = false;
```

Comment:
Curious if it would be more intuitive if this option is the opposite and by default inputs don't receive displacement. So this option could be named "Apply Displacement" or "Move With Ocean Surface" or something. I think a lot of the time inputs are added to objects like boats or static objects where this _applyDisplacementCorrection will be wanted. Thoughts?

Reply:
Hmmm yeah I guess so! Could call it "follow horizontal motion"?

Reply:
That sounds good